packages/mobile_app/lib/components/prophecy.dart

```diff
@@ -6,65 +6,92 @@ import 'package:prophecy/bloc.dart';
 import 'package:prophecy_model/prophecy_model.dart';
 import 'prophecy_record.dart';
 import 'prophecy_is_loading.dart';
-import 'package:app/screens/loading.dart';
 import 'package:flutter_svg/flutter_svg.dart';
 import 'package:mutable_wrappers/mutable_wrappers.dart';
 import 'package:users_repository/users_repository.dart';
 import 'package:language/language.dart';
 
 import 'package:int_datetime/int_datetime.dart';
-import 'package:algorithm/astro.dart' show BasicAstrology;
+import 'package:algorithm/algorithm.dart';
 
 class Prophecy extends StatelessWidget {
+  final int dt;
   final UserEntity user;
   final labelStr;
   final birthRow = MutableObject(null);
-  Prophecy({@required this.user})
+  Prophecy({@required this.user, @required this.dt})
```

Comment:
what is `dt`? can you please add a comment or a dartdoc on a constructor?

Reply:
I will do.
dt stands for date and time that saved in seconds or miliseconds since Epoch.
It is used in most APIs by a name "dt".